utilities/user_utilities.py

```diff
@@ -1149,6 +1149,31 @@ def get_admin_group_ids():
         __logger.error("Groups not retrievable")
         return []
 
+def overwrite_subevents_to_superevent(overwrite_subevent_list, super_eventid):
```

Comment:
I think it might be better to add a short comment to describe the logic in this method. Thanks.